javaee-websocket-gwt-rpc-sample/pom.xml

```diff
@@ -36,7 +36,7 @@
 	<build>
 		<plugins>
 			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
+				<groupId>${gwt.plugin.groupId}</groupId>
```

Comment:
..okay, this miiight change, but the whole plugin block has to be rewritten if it does.